scene/resources/packed_scene.cpp

```diff
@@ -485,6 +518,7 @@ Error SceneState::_parse_node(Node *p_owner, Node *p_node, int p_parent_idx, Map
 	p_node->get_property_list(&plist);
 
 	Array pinned_props = _sanitize_node_pinned_properties(p_node);
+	Dictionary missing_resource_properties = p_node->get_meta(META_MISSING_RESOURCES, Dictionary());
 
 	for (const PropertyInfo &E : plist) {
 		if (!(E.usage & PROPERTY_USAGE_STORAGE)) {
```

Comment:
Nodes report their theme override properties as having `PROPERTY_USAGE_STORAGE` based on whether or not they're actually set. If a theme override property was set to a missing resource, it will be reported as not having `PROPERTY_USAGE_STORAGE` and it will be skipped. This should check if `missing_resource_properties` has the property.